adoc/chapters/programming_interface.adoc

```diff
@@ -16937,8 +16937,8 @@ The vector supports member functions that allow construction of a new vector
 from a swizzled set of component elements.
 
 [code]#vec<typename _DataT_, int _NumElements_># is a vector type that compiles
-down to a <<backend>> built-in vector types on SYCL devices, where possible, and
-provides compatible support on the host or when it is not possible.
+down to a <<backend>> built-in vector type on SYCL devices, where possible, and
+provides compatible support on the host when not possible.
```

Comment:
This entire sentence seems like an implementation detail to me.  How could you write a test for this sentence?  It just says that it might or might not use a built-in vector type in device code.

The part about "provides compatible support on the host when not possible" seems not quite accurate.  The `vec` type works on the host regardless of how `vec` is implemented in device code.  So, I think this part of the sentence is just saying that `vec` is also available in host code.  We say that already in the first sentence of the first paragraph in this section.

I think we should just delete the sentence, rather than trying to fix it.

Reply:
Thanks.  Can you update the description of this PR also?